src/xdocs/writingchecks.xml

```diff
@@ -523,45 +522,44 @@ public class MethodLimitCheck extends AbstractCheck
         MethodLimitCheck module in the configuration file. It doesn&#39;t get any
         simpler than that. The secret is that Checkstyle uses
         <a href="https://docs.oracle.com/javase/tutorial/reflect/member/fieldValues.html">
-        JavaBean reflection to set the JavaBean properties</a>. That works for all
-        primitive types like boolean, int, long, etc., plus Strings, plus arrays of these types.
+        JavaBean reflection</a> to set the JavaBean properties. This works for all
+        primitive types like boolean, int, long, etc., Strings, and arrays of these types.
```

Comment:
The `etc., ` and continuing on to list things is not appropriate. `etc.` should be used at the end of the list, not in the middle as it is meant to cover everything else.

Reply:
the `etc.` here was the ending of the list of primitives, but not of the list of properties. Changed to:

`for all primitive types (for example boolean, int, and long), Strings and arrays of these types.`